api/v1/helpers.go

```diff
@@ -648,6 +648,18 @@ func (v *VerticaDB) IsKnownDepotVolumeType() bool {
 	return false
 }
 
+// IsDepotVolumeManaged returns true if vertica.com/disable-depot-volume-management
+// is false or not set
+func (v *VerticaDB) IsDepotVolumeManaged() bool {
+	ndvma, found := v.Annotations[vmeta.NoDepotVolumeManagementAnnotation]
+	if !found || ndvma == vmeta.NoDepotVolumeManagementAnnotationFalse ||
+		ndvma == "" {
+		return true
+	}
+
+	return false
```

Comment:
Move this logic to annotations.go. Something like:
```
func DisableDepotVolumeManagement(annotations map[string]string) bool {
	return lookupBoolAnnotation(annotations, NoDepotVolumeManagementAnnotation, NoDepotVolumeManagementAnnotationFalse)
}
``

Reply:
will do that tomorrow :) will also provide an example setup.

Reply:
preparing the example, I will attach it.